<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IOModuleClass</class>
 <widget class="QMainWindow" name="IOModuleClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1387</width>
    <height>883</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IOModule</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="rsWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="widget">
       <attribute name="title">
        <string>2D</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,20,1">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="0,5,1">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Path : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="PathEdit_2d">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="LoadBtn_2d">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Load</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QFrame" name="Viewer_2d">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 0, 0);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="5,1,1">
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>368</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="CapSaveBtn_2d">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Save (Capture)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="SaveBtn_2d">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Save (Origin)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget_3">
       <attribute name="title">
        <string>CAD</string>
       </attribute>
       <widget class="QFrame" name="Viewer_cad_1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>1301</width>
          <height>571</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>531</width>
          <height>61</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="ViewVertex_1">
           <property name="text">
            <string>Vertex</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ViewWireFrame_1">
           <property name="text">
            <string>WireFrame</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ViewFace_1">
           <property name="text">
            <string>Face</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ViewEdgedFace_1">
           <property name="text">
            <string>EdgedFace</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>40</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Path : </string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="PathEdit_cad_1">
        <property name="geometry">
         <rect>
          <x>54</x>
          <y>2</y>
          <width>1161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="LoadBtn_cad_1">
        <property name="geometry">
         <rect>
          <x>1220</x>
          <y>0</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>136</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
       <widget class="QSplitter" name="splitter_3">
        <property name="geometry">
         <rect>
          <x>980</x>
          <y>660</y>
          <width>431</width>
          <height>41</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSplitter" name="splitter_2">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>660</y>
          <width>431</width>
          <height>41</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSplitter" name="splitter_1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>690</y>
          <width>431</width>
          <height>41</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1387</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>LoadBtn_2d</tabstop>
  <tabstop>CapSaveBtn_2d</tabstop>
  <tabstop>SaveBtn_2d</tabstop>
  <tabstop>PathEdit_cad_1</tabstop>
  <tabstop>LoadBtn_cad_1</tabstop>
  <tabstop>ViewVertex_1</tabstop>
  <tabstop>ViewWireFrame_1</tabstop>
  <tabstop>ViewFace_1</tabstop>
  <tabstop>ViewEdgedFace_1</tabstop>
  <tabstop>PathEdit_2d</tabstop>
 </tabstops>
 <resources>
  <include location="iomodule.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>slotTabChanged(int)</slot>
 </slots>
</ui>
